### PR TITLE
Norwegian translation

### PR DESCRIPTION
**Pull request by phillipj**
Hi,

we're still at work with translation some of the nodeschool workshops to norwegian. This is pretty much the same PR which got merged into workshopper a couple of days ago workshopper/workshopper#115.

@leftieFriele @Torsph could one of you review & confirm these changes?

### CODE REVIEW

**Comment by tgfjt** [comment]
Hi :smile:  @phillipj 
Could you replace  `ノルウェーの` with `ノルウェー語` on `i18n/ja.json`? 

because:
-  `ノルウェーの` means "Norwegian (something)"
-  `ノルウェー語` means "Norwegian (language)"

File: i18n/de.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "vi": "Vietnamesisch (Tiếng Việt)",
     "ru": "Russisch (Русский)",
     "ko": "Koreanisch (한국어)",
-    "pl": "Polnisch (Polski)"
+    "pl": "Polnisch (Polski)",
+    "nb-no": "Norwegisch (Norsk)"
```
Comment (Torsph) [inline i18n/de.json:23]: If I should be picky. Should it stand "Norsk Bokmål"?

File: i18n/footer/nb-no.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ __»__ For å skrive ut oppgaveteksten igjen kjør: `{appname} print`
```
Comment (Torsph) [inline i18n/footer/nb-no.md]: Do we need to write "kjør" ?

File: i18n/nb-no.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    "pt-br": "Brisiliansk Potugisisk (Português)",
+    "vi": "Vietnamesisk (Tiếng Việt)",
+    "ru": "Russisk (Русский)",
+    "ko": "Koreansk (한국어)",
+    "pl": "Polsk (Polski)",
+    "nb-no": "Norsk (Norsk)"
+  },
+  "error": {
+    "exercise": {
+      "none_active": "Ingen valgte oppgaver. Velg en fra menyen.",
```
Comment (Torsph) [inline i18n/nb-no.json]: Maybe: "Ingen oppgave valgt. Velg en fra menyen"

File: i18n/nb-no.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      "missing": "Fant ikke oppgaven: {{{name}}}",
+      "unexpected_error": "Kunne ikke {{mode}}: {{{err}}}",
+      "missing_file": "FEIL: {{{exerciseFile}}} finnes ikke!",
+      "not_a_workshopper": "FEIL: {{{exerciseFile}}} er ikke en workshopper oppgave",
+      "preparing": "Feil ved oppsett av oppgave: {{{err}}}",
+      "loading": "Feil ved lasting av oppgave: {{{err}}}"
+    },
+    "no_uncomplete_left": "Alle oppgaver er utført etter du har gjennomført denne oppgaven",
+    "cleanup": "Feil under opprensking: {{{err}}}",
```
Comment (Torsph) [inline i18n/nb-no.json]: "Opprydding" maybe? I dont really care :)

File: i18n/de.json (hunk 1 above)
Comment (phillipj) [inline i18n/de.json:23]: To be 100% correct, yes. IMO it wont have much benefits before someone adds nn-no (nynorsk).

Comment (Torsph) [inline i18n/de.json:23]: I agree :+1:

**Comment by phillipj** [comment]
Thanks @tgfjt, absolutely!

**Comment by phillipj** [comment]
Appreciate the feedback, just pushed some changes. I'm all done if nothing else seems odd to anyone

**Comment by Torsph** [comment]
Im good!

**Comment by martinheidegger** [comment]
Thanks!

**Pull request merged**